FusionIIIT/applications/academic_procedures/views.py

```diff
@@ -57,23 +57,23 @@ def academic_procedures(request):
     current_user = get_object_or_404(User, username=request.user.username)
     # print("current username : ")
     # print(current_user)
-    print("current user id : request.user.id")
+    # print("current user id : request.user.id")
     # print(request.user.id)
-    print("user detail id : request.user")
+    # print("user detail id : request.user")
     # print(request.user)
 
     #mohit-these are extra info details , user id used as main id
-    user_details = ExtraInfo.objects.get(user = request.user)
+    user_details = ExtraInfo.objects.select_related().get(user = request.user)
```

Comment:
For example, here.

Reply:
Also, remove the print unnecessary print statements as well.

Reply:
Done